packages/web-components/fast-foundation/src/anchor/anchor.ts

```diff
@@ -105,6 +105,33 @@ export class Anchor extends FoundationElement {
      * References the root element
      */
     public control: HTMLAnchorElement;
+
+    /**
+     * @internal
+     */
+    public connectedCallback(): void {
+        super.connectedCallback();
+
+        this.handleUnsupportedDelegatesFocus();
+    }
+
+    /**
+     * Overrides the focus call for FireFox where delegatesFocus is unsupported.
+     * This should be removed as soon as FireFox supports delegatesFocus properly,
+     * the check for override does not work for any browser outside of FireFox.
+     * Relevant PR on the Firefox browser: https://phabricator.services.mozilla.com/D123858
+     */
+    private handleUnsupportedDelegatesFocus = () => {
+        // InstallTrigger is only defined in FireFox
+        if (
+            typeof ((window as unknown) as WindowWithInstallTrigger).InstallTrigger !==
+            "undefined"
+        ) {
```

Comment:
Since this only affects buttons/anchors when `delegatesFocus` doesn't work, we could just check to see if `delegatesFocus` is set and apply the focus regardless of the support:

```suggestion
        if (this.$fastController.definition.shadowOptions?.delegatesFocus) {
```

While `shadowRoot.delegatesFocus` may not be available in `closed` mode, the `$fastController` definition will always be available regardless of the mode used.

Reply:
Yea, but I don't see how the $fastController helps us. I saw that you can see the shadowOptions that were passed in, but that doesn't help us because the option passed in is the same. Is there a property in $fastController we can use?

Reply:
Isn't `$fastController.definition.shadowOptions?.delegatesFocus` our construct and could still be true even if Firefox ignores the setting?

Reply:
Yes. That's whatever config is passed when the component is defined. It doesn't get altered based on platform capabilities.

Reply:
Another potential check could be to see if `window.ShadowRoot?.prototype.hasOwnProperty("delegatesFocus")` - Firefox and Safari 14 return `false`, and Chrome and Edge return `true`.

Reply:
That sounds good to me, I can make that change if we're confident in it, though I'd rather not churn on this PR much more if we've already got a working solution for FireFox.

Reply:
Thanks for working through this all and finding a solid solution that will work cross browser!